packages/core/src/components/DropDownMenu/DropDownMenu.tsx

```diff
@@ -145,7 +145,7 @@ export const HvDropDownMenu = (props: HvDropDownMenuProps) => {
         >
           {icon || (
             <MoreOptionsVertical
-              role="presentation"
+              aria-hidden
```

Comment:
`role="none"` [would also fix the "issue"](https://github.com/lumada-design/hv-uikit-react/pull/3632), even though it's supposed to be exactly [the same as `presentation`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/presentation_role) 

This is most likely a bug in `pa11y`/axe. The `svg` has no semantic children.